README.md

```diff
@@ -117,6 +117,18 @@ Save this file in the root folder of your project (e.g. where the package.json
 file is found). You can also save it to the user home directory if you want to
 share a global config between different projects.
 
+If you have a project inside a project with both containing a `package.json` file,
+you can specify that the subproject is not root like this:
```

Comment:
Because most of the configuration for import JS is in the import JS configuration file, I think it would be helpful to make it clearer that this snippet is showing `package.json`. Maybe all we need to do is add "...not root like this in `package.json`:" or something

Reply:
done